site/content/en/docs/tasks/manage/enable_kubeviz.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Enable KueueViz"
+date: 2024-01-15
+weight: 3
+description: >
+  Installing and configuring KueueViz, a web-based visualization tool for Kueue workload monitoring.
+---
+
+KueueViz is a web-based visualization tool that provides real-time monitoring of Kueue workloads, queues, and resource allocation. It offers an intuitive dashboard for observing job queue status, resource utilization, and workload progression.
+
+This page shows how to install and configure KueueViz in your cluster.
+
+The page is intended for [batch administrators](/docs/tasks#batch-administrator).
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running
+- The kubectl command-line tool has communication with your cluster
+- The Helm command-line tool is installed
+- (Optional) Kueue is installed in your cluster
+- (Optional) An ingress controller for external access (e.g. Nginx Ingress Controller)
+
+KueueViz can be installed using Helm, kubectl, or Kustomize. Choose the method that best fits your workflow.
```

Comment:
```suggestion
KueueViz can be installed using Helm (recommended), or kubectl. Choose the method that best fits your workflow.
```
wdyt? As mentioned in the other commit, I don't think we need to promote kustomize approach in the docs. I want to encourage users with special needs to add more parameters to Helm installation, rather than to hack them via kustomize.